helpers/clickClearRefinements.ts

```diff
@@ -5,11 +5,20 @@ declare namespace WebdriverIOAsync {
 }
 
 browser.addCommand('clickClearRefinements', async () => {
+  const oldUrl = await browser.getUrl();
   const clearButton = await browser.$(`.ais-ClearRefinements-button`);
   // Assures us that the element is in the viewport
   await clearButton.scrollIntoView();
 
   await clearButton.click();
 
-  return browser.waitForElement(`.ais-ClearRefinements-button--disabled`);
+  await browser.waitForElement(`.ais-ClearRefinements-button--disabled`);
+
+  // Changing the URL will also change the page element IDs in Internet Explorer
+  // Not waiting for the URL to be properly updated before continuing can make the next tests to fail
```

Comment:
```suggestion
  // Not waiting for the URL to be properly updated before continuing can make the next tests fail
```

This would need to be updated in other locations.